tests/test-cases/readers/cube-config/v1.3/qudt-unit-template-testing.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Where is this being used?

Reply:
This is needed for the "Successfully outputs a cube using schema v1.3" behave test in cube.feature file.